constants/profile.tsx

```diff
@@ -0,0 +1,172 @@
+  "Classical Languages and Literatures",
+  "Climate and Meteorology",
+  "Cognitive Science",
+  "Communication and Media",
+  "Community and Global Public Health",
+  "Comparative Culture and Identity",
+  "Comparative Literature",
+  "Composition",
+  "Computer Engineering",
+  "Computer Science",
+  "Creative Writing and Literature",
+  "Dance",
+  "Data Science",
+  "Dental Hygiene",
+  "Drama",
+  "Earth and Environmental Sciences",
+  "Ecology, Evolution, and Biodiversity",
+  "Economics",
+  "Electrical Engineering",
+  "Elementary Teacher Education",
+  "Engineering Physics",
+  "English",
+  "Environment",
+  "Environmental Engineering",
+  "Ethnic Studies",
+  "Evolutionary Anthropology",
+  "Film, Television, and Media",
+  "French and Francophone Studies",
+  "Gender and Health",
+  "General Studies",
+  "German",
+  "Global Environment and Health",
+  "Greek (Ancient) Language and Literature",
+  "Greek (Modern) Language and Culture",
+  "History",
+  "History of Art",
+  "Industrial and Operations Engineering",
+  "Informatics",
+  "Information",
+  "Interarts Performance",
+  "Interdisciplinary Astronomy",
+  "Interdisciplinary Chemical Sciences",
+  "Interdisciplinary Physics",
+  "International Security, Norms, and Cooperation",
+  "International Studies",
+  "Italian",
+  "Jazz & Contemporary Improvisation",
+  "Judaic Studies",
+  "Latin American and Caribbean Studies",
+  "Latin Language and Literature",
+  "Latina/Latino Studies",
+  "Linguistics",
+  "Materials Science and Engineering",
+  "Mathematical Sciences",
+  "Mathematics",
+  "Mathematics of Finance and Risk Management",
+  "Mechanical Engineering",
+  "Microbiology",
+  "Middle East Studies",
+  "Middle Eastern and North African Studies",
+  "Molecular, Cellular, and Developmental Biology",
+  "Movement Science",
+  "Music",
+  "Music Education",
+  "Music Theory",
+  "Musical Theatre",
+  "Musicology",
+  "Naval Architecture and Marine Engineering",
+  "Neuroscience",
+  "Nuclear Engineering and Radiological Sciences",
+  "Nursing",
+  "Organ",
+  "Organizational Studies",
+  "Performing Arts Technology",
+  "Pharmaceutical Sciences",
+  "Philosophy",
+  "Philosophy, Politics, and Economics",
+  "Physics",
+  "Piano",
+  "Plant Biology",
+  "Polish",
+  "Political Economy and Development",
+  "Political Science",
+  "Psychology",
+  "Public Health Sciences",
+  "Public Policy",
+  "Pure Mathematics",
+  "Romance Languages and Literatures",
+  "Russian",
+  "Russian, East European, and Eurasian Studies",
+  "Secondary Teacher Education",
+  "Social Theory and Practice",
+  "Sociology",
+  "Space and Science Engineering",
+  "Spanish",
+  "Sport Management",
+  "Statistics",
+  "Strings",
+  "Structural Biology",
+  "Theatre & Drama",
+  "Program",
+  "Undecided",
+  "Urban Technology",
+  "Voice",
+  "Winds and Percussion",
+  "Women's and Gender Studies",
+].map((x) => ([x, x])));
+
+export {
+  Year, RoleType, Interest, FieldOfStudy,
+};
```

Comment:
Great idea to have them in `/constants`
Thank you for all the work that went into this PR @elliotfklein ! 🥳